modules/data/src/main/java/io/fluo/webindex/data/fluo/InlinksObserver.java

```diff
@@ -81,29 +81,32 @@ public void process(TransactionBase tx, Bytes row, Column col) throws Exception
       }
     }
 
-    Long incount = ttx.get().row(row).col(FluoConstants.PAGE_INCOUNT_COL).toLong(0) + change;
-    if (incount <= 0) {
-      if (incount < 0) {
-        log.error("Incount for {} is negative: {}", row, incount);
-      }
-      ttx.mutate().row(row).col(FluoConstants.PAGE_INCOUNT_COL).delete();
-    } else {
-      ttx.mutate().row(row).col(FluoConstants.PAGE_INCOUNT_COL).set(incount);
+    boolean knownPage = true;
```

Comment:
A comment about what a knownPage is would be helpful... I think I know from discussion.   I am thinking its for the case where a page has inbound links, but was not in the crawled set?